daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala

```diff
@@ -249,24 +250,49 @@ private[lf] final class CommandPreprocessor(
     speedy.Command.LookupByKey(templateId, key)
   }
 
+  private[preprocessing] def unsafeResolveTyConName(
+      pkgPriority: Map[Ref.PackageName, Ref.PackageId],
```

Comment:
why is this argument called`pkgPriority` here and `pkgMap` elsewhere?

Reply:
good catch.